cpg_workflows/stages/gatk_sv/gatk_sv_multisample_2.py

```diff
@@ -87,11 +87,11 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
             for batch_name in batch_names
         ]
         bincov_files = [
-            batch_prefix / batch_name / 'GatherBatchEvidence' / 'RD.txt.gz'
+            batch_prefix / batch_name / 'GatherBatchEvidence' / f'{self.name}.RD.txt.gz'
```

Comment:
Does this need to match the RD filename from _gatk_sv_multisample_1.py_?

Reply:
Well... that's an entirely fair point. Methinks a follow-up to change this is needed 🤦‍♂️